scss/_grid.scss

```diff
@@ -5,18 +5,52 @@
 @if $enable-grid-classes {
   .row {
     @include make-row();
+
+    > * {
+      @include make-col-ready();
+    }
   }
 
-  // Remove the negative margin from default .row, then the horizontal padding
-  // from all immediate children columns (to prevent runaway style inheritance).
-  .no-gutters {
-    margin-right: 0;
-    margin-left: 0;
+  ul,
+  ol {
+    // stylelint-disable-next-line selector-no-qualifying-type
+    &.row {
+      @include list-unstyled();
```

Comment:
I'm worried whether this reset should be included. This might be unexpected behavior and I don't know how often this will be used. It just be used only one location in the new document.

If it's not the core of this PR, I prefer discuss it in a separate issue.

Reply:
I had suggested the scoping of these at least to `ul, ol` earlier in the PR. It makes sense if we want folks to be using this kind of markup as a shortcut, but we could just as easily ask folks to use `.list-unstyled` in addition to `.row`.

Reply:
I missed [the comment](https://github.com/twbs/bootstrap/pull/28517#discussion_r364956887). The `ul.row` raises the CSS priority, so for example overwriting its `padding-left` may cause confusion.

```html
<ul class="row custom-class">...</ul>
```
```scss
.custom-class {
   padding: 1rem; // left padding is not apply
}
```

It is a safe way for users to add `.list-unstyled` deliberately. Or, I think as in the first @MartijnCuppens 's suggestion, applying rest list style to `.row` without limiting to `ul` and `li` may not cause confusion of CSS priority. e.g.:

```css
.row {
  ...
  padding: 0;
  list-style: none;
}
```

Reply:
Good call, I think it'd be better to ask folks to add the `.list-unstyled` to make this work.

Reply:
Good call, I think it'd be better to ask folks to add the `.list-unstyled` to make this work.

Reply:
Indeed, removed it.